tox.ini

```diff
@@ -1,36 +1,21 @@
 [tox]
```

Comment:
I think this tox.ini file should be removed (or renamed). If tox.ini is present, robertdebock's action ignores the image matrix we've specified, as you can see from the test results; they're all running against the default 18.04 image.

Reply:
Remove tox.ini